trigger-openqa_in_openqa

```diff
@@ -29,14 +29,13 @@ find_latest_published_tumbleweed_image() {
         echo "Unable to find latest published Tumbleweed build."
         exit 1
     fi
-    qcow=$(/usr/share/openqa/script/openqa-cli api --host "${tw_openqa_host}" assets get \
+    qcow=$(${openqa_cli} api --host "${tw_openqa_host}" assets get \
         | jq -r "[.assets[] | select(.name | test(\"Tumbleweed-$arch-$latest_published_tw_build-Tumbleweed\\\\@$machine.qcow\"))] | .[0] | .name")
-    if ! [[ $qcow ]]; then
+    if [[ "$qcow" = "null" ]]; then
```

Comment:
What about the case when the JSON is completely invalid? Maybe `$qcow` is still empty then. I also remember that I've tested this case and maybe in some cases `$qcow` is really just empty. I'd simply check for both (null and empty) here to be sure.

Reply:
In case the JSON is invalid, jq and the script should abort because of `bash -e`.
But also checking for the empty string doesn't sound like a bad idea.

Reply:
If the JSON is invalid there's no return value:

    parse error: Unfinished JSON term at EOF at line 2, column 0

I can't come up with a case where the return value is the empty string.

Reply:
this change makes sense to me. In 9a1ae1d mkittler changed from "sed" to "jq" so instead of an empty string we now get the null value from jq